### PR TITLE
Update BaseComponent.ts

### PR DESCRIPTION
**Pull request by jgeddes-coveo**
Small grammar fixes to docs

[Deploy](https://dashboard.heroku.com/pipelines/a3535101-5bbf-4a5b-a909-47fcf8c9f149)

### CODE REVIEW

**Review by ThibodeauJF — APPROVED** [review]

**Comment by ThibodeauJF** [comment]
I will merge it for you 😄

File: src/ui/Base/BaseComponent.ts [hunk 1]
```diff
@@ -22,8 +22,8 @@ export class BaseComponent {
    */
   public disabled = false;
   /**
-   * The static ID that each component need to be identified.<br/>
-   * For example, SearchButton -> static ID : SearchButton -> className : CoveoSearchButton
+   * The static ID that each component needs to be identified.<br/>
```
Comment (ThibodeauJF) [inline src/ui/Base/BaseComponent.ts:25]: `needs to be identified with` ?

**Comment by btaillon** [comment]
Is there a JIRA for this?

File: src/ui/Base/BaseComponent.ts (hunk 1 above)
Comment (jgeddes-coveo) [reply src/ui/Base/BaseComponent.ts:25]: I guess it depends: is it the static ID that each component needs in order to be identified? Or is it the static ID that identifies each component?

Comment (ThibodeauJF) [reply src/ui/Base/BaseComponent.ts:25]: Hmm the first, could we add `in order` in the string, would sound more clear a bit

Comment (samisayegh) [reply src/ui/Base/BaseComponent.ts:25]: Another option could be to reorder the words: `The static ID that identifies each component`.

@jgeddes-coveo, both those phrasings seem similar, I'm not catching the nuance? This property has value `Tab` for the tab component, `Facet` for the facet component, and so on.

**Review by samisayegh — APPROVED** [review]

**Comment by samisayegh** [comment]
Close this PR because the change is not too significant and some of the checks are currently stuck (this is due to a change I made part of mirgrating the repo to a newer CI/CD version).

**Pull request closed without merging**